src/components/dropdown/react/Dropdown.tsx

```diff
@@ -64,12 +85,63 @@ const DEFAULT_PROPS: IDefaultPropsType = {};
 const Dropdown: React.FC<DropdownProps> = ({
     children,
     className = '',
-    ...props
-}: DropdownProps): React.ReactElement => (
-    <div className={classNames(className, handleBasicClasses({ prefix: CLASSNAME }))} {...props}>
-        {children}
-    </div>
-);
+    closeOnClick = DEFAULT_PROPS.closeOnClick,
+    escapeClose = DEFAULT_PROPS.escapeClose,
+    offset,
+    // overToggle = DEFAULT_PROPS.overToggle,
```

Comment:
Remove if useless

Reply:
I'll remove for now but as discussed in the comment, Guillaume is going to add the option to the popover and I'll add it here as well once it's done.